applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -628,16 +566,23 @@ int streamctrl_start(void)
 	ret = zbus_subscribers_create();
 	ERR_CHK_MSG(ret, "Failed to create zbus subscriber threads");
 
-	ret = bt_rend_init();
-	ERR_CHK(ret);
+	ret = zbus_link();
+	ERR_CHK_MSG(ret, "Failed to link zbus producers and observers");
 
-	ret = bt_content_ctrl_init();
-	ERR_CHK(ret);
+	ret = audio_datapath_thread_create();
+	ERR_CHK_MSG(ret, "Failed to create audio datapath thread");
 
-	ARG_UNUSED(le_audio_rx_data_handler);
+	ret = broadcast_sink_enable(le_audio_rx_data_handler);
+	if (ret) {
+		LOG_WRN("Failed to enable broadcast sink");
```

Comment:
Could be ERR I guess, same for the one below

Reply:
Changed to ERR_CHK_MSG